pkg/asset/manifests/openshift.go

```diff
@@ -129,6 +129,12 @@ func (o *Openshift) Generate(dependencies asset.Parents) error {
 		if err != nil {
 			return err
 		}
+
+		// We need to replace the local cacert path with one that is used in OpenShift
+		if cloud.CACertFile != "" {
+			cloud.CACertFile = "/etc/kubernetes/static-pod-resources/configmaps/cloud-config/ca-bundle.pem"
```

Comment:
Will this break [Terraform variable generation][1]?  I'd be happier with a new asset, or asset property, that read the local CA cert file from the user-configured location into the asset store, so we didn't have to read it in dynamically durring Terraform variable generation.  Then we wouldn't have to worry about whether clobbering the value here would have an impact on that behavior or not.

[1]: https://github.com/openshift/installer/blob/7a772518015fc14b48426344e8b3800b16b50d15/pkg/asset/cluster/tfvars.go#L384-L385